specs/gloas/p2p-interface.md

```diff
@@ -297,6 +327,37 @@ The following validations MUST pass before forwarding the
 - _[REJECT]_ `signed_execution_payload_bid.signature` is valid with respect to
   the `bid.builder_index`.
 
+###### `proposer_preferences`
+
+*[New in Gloas:EIP7732]*
+
+This topic is used to propagate signed proposer preferences as
+`SignedProposerPreferences`. These messages allow validators to communicate
+their preferred `fee_recipient` and `gas_limit` to builders.
+
+The following validations MUST pass before forwarding the
+`signed_proposer_preferences` on the network, assuming the alias
+`preferences = signed_proposer_preferences.message`:
+
+- _[IGNORE]_ `preferences.proposal_slot` is in the next epoch -- i.e.
+  `compute_epoch_at_slot(preferences.proposal_slot) == get_current_epoch(state) + 1`.
```

Comment:
wouldn't it be better if we also allow to broadcast preferences for the current epoch, eg. if a node was offline for whatever reason it still get a chance to submit the preference